ext/openssl/php_openssl.h

```diff
@@ -86,6 +94,13 @@ php_stream_transport_factory_func php_openssl_ssl_socket_factory;
 
 void php_openssl_store_errors();
 
+PHP_OPENSSL_API int php_openssl_cipher_iv_length(char *method, zend_long *ret);
```

Comment:
I think it would be better to return the IV or negative value if there is an error on unknown method?